internal/experiment/webconnectivityqa/control.go

```diff
@@ -44,6 +44,7 @@ func controlFailureWithSuccessfulHTTPWebsite() *TestCase {
 		ExpectErr: false,
 		ExpectTestKeys: &testKeys{
 			ControlFailure: "unknown_failure: httpapi: all endpoints failed: [ connection_reset; connection_reset; connection_reset; connection_reset;]",
+			DNSConsistency: "consistent",
```

Comment:
This is not a regression. Web Connectivity LTE performs TLS connections with the website even when the URL is HTTP to check whether we can consider the IP addresses as valid. Because TLS succeeds, we end up considering the DNS as consistent.